Example/ViewController.swift

```diff
@@ -540,21 +540,38 @@ extension ViewController: NavigationViewControllerDelegate {
         dismissActiveNavigationViewController()
         clearNavigationMapView()
     }
-    
-    // MARK: - Offline Navigation Tiles switching
+}
+
+// MARK: - Offline Navigation Tiles switching
+
+extension ViewController {
     
     func tilesVersioning(wantsSwitchTilesVersion versions: [String], completion: @escaping (String?) -> ()) {
         let alertController = UIAlertController(title: "Switch to Offline Regions",
                                                 message: "There are next tiles versions available:",
                                                 preferredStyle: .actionSheet)
         
-        versions.forEach{ version in
-            alertController.addAction(UIAlertAction(title: version, style: .default, handler: { _ in
+        versions.forEach { version in
+            // converting version name format from "2020_01_02-03_00_00" -> "2020/01/02 03:00:00"
+            let parts = version.split(separator: "-", maxSplits: 1, omittingEmptySubsequences: true)
+            let versionName = parts.first!.replacingOccurrences(of: "_", with: "/") + " " +
```

Comment:
Maybe it's better to parse this into a `Date` and show a localized string from formatted?

Reply:
I removed the naming piece, so it's not relevant anymore.